sdk/program/src/stake/mod.rs

```diff
@@ -5,8 +5,3 @@ pub mod state;
 pub mod program {
     crate::declare_id!("Stake11111111111111111111111111111111111111");
 }
-
-/// The minimum stake amount that can be delegated, in lamports.
-/// NOTE: This is also used to calculate the minimum balance of a stake account, which is the
-/// rent exempt reserve _plus_ the minimum stake delegation.
-pub const MINIMUM_STAKE_DELEGATION: u64 = 1;
```

Comment:
Unfortunately this is subject to the backwards compatibility policy and should be deprecated instead

Reply:
Done!